src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetHINSTANCETests.cs

```diff
@@ -36,10 +36,17 @@ public void GetHINSTANCE_NullModule_ThrowsArgumentNullException()
         }
 
         [Fact]
-        [ActiveIssue(30925)]
-        public void GetHINSTANCE_NonRuntimeModule_ThrowsArgumentNullException()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void GetHINSTANCE_NonRuntimeModule_Returns_IntPtrMinusOne()
         {
-            AssertExtensions.Throws<ArgumentNullException>("m", "Module must be a runtime Module object.", () => Marshal.GetHINSTANCE(new NonRuntimeModule()));
+            Assert.Equal((IntPtr)(-1), Marshal.GetHINSTANCE(new NonRuntimeModule()));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public void GetHINSTANCE_NonRuntimeModule_ThrowsArgumentNullException_NetFramework()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("m", () => Marshal.GetHINSTANCE(new NonRuntimeModule()));
```

Comment:
.NET Framework is incorrectly using the "Module must be a runtime Module object." message as the argument name. 

This may rather be just: `Assert.Throws<ArgumentNullException>(() => Marshal.GetHINSTANCE(new NonRuntimeModule()));`